src/data/markdown/docs/30 xk6-browser/30 xk6-browser.md

```diff
@@ -55,9 +72,13 @@ export default function () {
 If you have downloaded the pre-built binary you will find the binary named `xk6-browser`. Using the [example](#example), create a new file (such as `browser_test.js`) in the same directory as the pre-built binary, and paste the example in that file. Now run:
 
 ```
-xk6-browser run browser_test.js
+./xk6-browser run browser_test.js
 ```
 
+> Note the `./` prefix, which tells your shell to run the binary located in the current working directory. This is required on macOS and Linux, but not on Windows if you're using the `cmd.exe` shell. If you're using PowerShell, you should specify `.\xk6-browser` instead.
+>
+> If you installed xk6-browser via a system package, or you've placed the binary in a directory that's part of your `$PATH` environment variable, then you can omit the `./` or `.\` prefix.
```

Comment:
Is this necessary information? My guess is that most people who want to browser test know this, especially if they know about $PATH variables.

But, I trust that the author knows the reader best.

I'm more open to leaving a note just for windows users, especially if `cmd.exe` and Powershell will fail with this syntax.

If we leave a note in, I suggest using the `<Blockquote>` Gatsby component to keep things consistent.

```
<Blockquote mod="note" title="A note for Windows users">

On Windows shells, the `./` file prefix will raise an exception.
Instead, try these:
- For `command.exe`, use no file prefix.
- In Powershell, use `.\xk6-browser`.

</Blockquote>
```

Will render as:

![image](https://user-images.githubusercontent.com/47385188/192040099-9f9fbe60-5cba-4171-b620-f7a42e002ba0.png)

Reply:
Resolved by 41c7e883.